tool/tsh/dbcmd.go

```diff
@@ -315,3 +323,20 @@ func (c *cliCommandBuilder) getMongoCommand() *exec.Cmd {
 	// fall back to `mongo` if `mongosh` isn't found
 	return exec.Command(mongoBin, args...)
 }
+
+func (c *cliCommandBuilder) getSQLServerCommand() *exec.Cmd {
+	args := []string{
+		// Host and port must be comma-separated.
+		"-S", fmt.Sprintf("%v,%v", c.host, c.port),
+		"-U", c.db.Username,
+		// Password is required by the client but doesn't matter as we're
+		// connecting to local proxy.
+		"-P", c.uid.New(),
```

Comment:
Should we do the same for Redis if MSSQL provides that option?

Reply:
I don't think it's needed since Redis clients use regular TLS handshake and connect to the web proxy endpoint already.